cloudbridge/cloud/providers/gce/services.py

```diff
@@ -1225,6 +1252,7 @@ def create(self, name, location=None):
                             .execute())
             if 'error' in response:
                 return None
+            time.sleep(2)
```

Comment:
This is not completely arbitrary. GCS has a rate limit of 1 operation per 2 seconds for bucket creation/deletaion: https://cloud.google.com/storage/quotas. So the 2s throttling is to avoid future failures.

Reply:
Ok, that makes sense. I think it would be great if we can add a comment in the code detailing that rationale, just so we don't trip up on its necessity in future.

Reply:
Done.